tests/test_extension.py

```diff
@@ -359,10 +368,25 @@ class TestComponent(Component):
         assert slot_call.component_id == "ca1bc3e"
         assert isinstance(slot_call.slot, Slot)
         assert slot_call.slot_name == "content"
+        assert isinstance(slot_call.slot_node, SlotNode)
+        assert slot_call.slot_node.template_name.endswith("test_extension.py::TestComponent")  # type: ignore
+        assert slot_call.slot_node.template_component == TestComponent
         assert slot_call.slot_is_required is True
         assert slot_call.slot_is_default is True
         assert slot_call.result == "Some content"
 
+    @djc_test(components_settings={"extensions": [SlotOverrideExtension]})
+    def test_on_slot_rendered__override(self):
```

Comment:
`on_slot_rendered` hook was missing a test for when a new string is returned from it.